component/core/src/main/java/org/exoplatform/social/core/jpa/search/ActivitySearchConnector.java

```diff
@@ -47,8 +47,10 @@ public class ActivitySearchConnector {
       "  \"query_string\":{" +
       "    \"fields\": [\"body\", \"posterName\"]," +
       "    \"default_operator\": \"AND\"," +
-      "    \"query\": \"@term@\"" +
-      "  }" +
+      "    \"query\": \"@term@~\"," +
```

Comment:
```suggestion
      "    \"query\": \"(@term@) OR (\\\"@phrase@\\\")^5\"," +
```
we may need to add pertinence/boost information for complete phrase match as well, WDYT ?

Reply:
> we may need to add pertinence/boost information for complete phrase match as well, WDYT ?

in that case, will phrase get the value of search term??

Reply:
ok, changed for full proposition as discussed, thanks.